php/laravel/README.md

```diff
@@ -0,0 +1,14 @@
+# Sentry PHP Laravel Example
+
+This shows how to use Sentry in Laravel to capture errors/exceptions
+
+# Setup
+1. `$ compose install`
```

Comment:
This should be `composer install` (and maybe without the `$` to prevent people copy pasting it.